vigranumpy/test/CMakeLists.txt

```diff
@@ -153,7 +153,7 @@ IF(NOT PYTHON_NOSETESTS_NOT_FOUND)
     # variable CMAKE_CFG_INTDIR contains a dot '.' for a Windows nmake build, or
     #                           '$(OutDir)' for a VisualStudio build (OutDir will be
     #                           set by VS at build time)
-    IF(NOT CMAKE_CFG_INTDIR STREQUAL ".")
+    IF(NOT CMAKE_GENERATOR MATCHES "^Visual Studio")
```

Comment:
Interestingly dropping the `NOT` is causing this to fail with NMake now. 😕

ref: https://ci.appveyor.com/project/jakirkham/vigra-feedstock/build/1.0.36/job/0u7xexy35t2awbak#L2980